tests/Unit/Cache/ResponseTagger/TraceableResponseTaggerTest.php

```
@@ -14,7 +14,7 @@ final class TraceableResponseTaggerTest extends TestCase
 {
     use ProphecyTrait;
 
-    private TraceableResponseTagger $collectTagsResponseTagger;
+    private TraceableResponseTagger $traceableResponseTagger;
```

PHPMD (2.15.0)

[warning] Avoid unused private fields such as '$traceableResponseTagger'. (undefined)

(UnusedPrivateField)